src/index.js

```diff
@@ -1,27 +1,36 @@
-import React from "react";
-import ReactDOM from "react-dom";
-import "./index.css";
-import App from "./App";
-import * as serviceWorker from "./serviceWorker";
-import store, { rrfProps } from "./store";
-import { ReactReduxFirebaseProvider } from "react-redux-firebase";
-import { Provider } from "react-redux";
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
+import store, { rrfProps } from './store';
+import { ReactReduxFirebaseProvider } from 'react-redux-firebase';
+import { Provider } from 'react-redux';
+import { createTheme } from '@material-ui/core';
 
-if ("serviceWorker" in navigator) {
+if ('serviceWorker' in navigator) {
   navigator.serviceWorker
-    .register("/firebase-messaging-sw.js")
-    .catch(function(err) {
-      console.log("Service worker registration failed, error:", err);
+    .register('/firebase-messaging-sw.js')
+    .catch(function (err) {
+      console.log('Service worker registration failed, error:', err);
     });
 }
 
+const theme = createTheme({
```

Comment:
is this `theme` variable used in the code? can you double check?

Reply:
Sorry this isn't used in the app currently. I added this to test the theme in the website